lib/config/config-file.js

```diff
@@ -18,7 +18,8 @@ const fs = require("fs"),
     pathIsInside = require("path-is-inside"),
     stripComments = require("strip-json-comments"),
     stringify = require("json-stable-stringify-without-jsonify"),
-    requireUncached = require("require-uncached");
+    requireUncached = require("require-uncached"),
+    Linter = require("../linter.js");
```

Comment:
I'm not sure using `Linter` here will work correctly when the generated config contains something like `extends` (i.e. when the user says they want to follow a popular styleguide). The second argument of `Linter#verifyAndFix` is supposed to be a "fully-resolved" config (i.e. it just has rules, without depending on any external plugins or `extends` clauses).

It might be better to use the [`CLIEngine#executeOnText`](https://eslint.org/docs/developer-guide/nodejs-api#cliengineexecuteontext) API instead. For example, maybe something like this would work:

```js
const results = new CLIEngine({ useEslintrc: false, ignore: false, baseConfig: config, fix: true })
    .executeOnText(content, ".eslintrc.js")

// ... extract output from results
```